jaggr-core/pom.xml

```diff
@@ -72,7 +72,7 @@
   <build>
     <resources>
       <resource>
-        <directory>..</directory>
+        <directory>${project.parent.basedir}</directory>
```

Comment:
Mars throws an exception on the ..  See http://stackoverflow.com/questions/33644244/import-maven-project-to-eclipse-getting-error